### PR TITLE
AQ-270 Trade History Screen

### PR DESCRIPTION
**Pull request by WillOrtiz7**
The purpose of this PR is to add the trade history screen which contains a reusable table component that can be used throughout the app.

Outline of changes:

- Users can access the trade history screen by pressing on the trade history option in the profile screen
- A table with infinite scroll is made with filler data

### CODE REVIEW

**Review by jasonmlam — COMMENTED** [review]
On my phone, there is about an inch of space between the header/body text and the table itself, which I think would look a little better if it was cut down a bit.

File: src/components/nested_screens/profile/TradeHistoryScreen.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+import HeaderContainer from "../../reusable_components/HeaderContainer";
+import CustomTable from "../../reusable_components/CustomTable";
 
 export default function TradeHistoryScreen() {
   return (
     <View style={styles.container}>
-      <Text style={styles.text}>Welcome to the trade history screen!</Text>
+      <HeaderContainer
+        headerText="Trade History"
+        bodyText="A detailed view of all transactions made by your job."
```
Comment (ryPattillo) [inline src/components/nested_screens/profile/TradeHistoryScreen.js:12]: "A detailed view of all transaction made by your job(s). "

Comment (WillOrtiz7) [reply src/components/nested_screens/profile/TradeHistoryScreen.js]: Addressed with latest commit

**Comment by WillOrtiz7** [comment]
> On my phone, there is about an inch of space between the header/body text and the table itself, which I think would look a little better if it was cut down a bit.

After updating the text to fix a typo there is a bit less space in between. I went ahead and made the amount of space that the header takes up a prop so when the component gets reused if the header only has a little bit of  body text it won't take up much space and when it has a lot of text it will take up more space.

**Review by jasonmlam — APPROVED** [review]

**Review by ryPattillo — APPROVED** [review]

**Pull request merged**